hack/benchmark/core/benchmark/strategy/bulk_insert.go

```diff
@@ -26,18 +26,56 @@ import (
 	"github.com/vdaas/vald/hack/benchmark/internal/core"
 )
 
+const (
+	maxBulkSize = 100000
+)
+
 func NewBulkInsert(opts ...StrategyOption) benchmark.Strategy {
 	return newStrategy(append([]StrategyOption{
 		WithPropName("BulkInsert"),
 		WithProp32(
 			func(ctx context.Context, b *testing.B, c core.Core32, dataset assets.Dataset, ids []uint, cnt *uint64) (interface{}, error) {
-				ids, errs := c.BulkInsert(dataset.Train())
+				size := func() int {
+					if maxBulkSize < dataset.TrainSize() {
+						return maxBulkSize
+					} else {
+						return dataset.TrainSize()
+					}
+				}()
+				v := make([][]float32, 0, size)
+				for i := 0; i < size; i++ {
+					arr, err := dataset.Train(i)
+					if err != nil {
+						break
+					}
+					v = append(v, arr.([]float32))
+				}
+				b.StartTimer()
+				defer b.StopTimer()
```

Comment:
I am sorry. if I make misstake.
```suggestion
		bb.StopTimer()
		bb.ReportAllocs()
		bb.ResetTimer()
		bb.StartTimer()

```

Reply:
The argument is **b *testing.B**, so your suggestion is typo, isn't it?

Reply:
sorry. My suggestion is typo.
b.〇〇Timer is correct:bow:

Reply:
Thank you! I fix it soon.